src/tests/ftest/util/daos_utils_base.py

```diff
@@ -62,6 +62,8 @@ def get_sub_command_class(self):
                 self.sub_command_class = self.SetAttrSubCommand()
             elif self.sub_command.value == "autotest":
                 self.sub_command_class = self.AutotestSubCommand()
+            elif self.sub_command.value == "version":
+                self.sub_command_class = self.VersionSubCommand()
```

Comment:
(pylint-no-member) Instance of 'PoolSubCommand' has no 'VersionSubCommand' member